lib/player.js

```diff
@@ -1640,9 +1643,11 @@ shaka.Player.prototype.usingEmbeddedTextTrack = function() {
  *
  * @param {shaka.extern.Track} track
  * @param {boolean=} clearBuffer
+ * @param {number=} safeMargin
```

Comment:
This deserves a comment for what this will do.  You should also include a part about browser compatibility for having this value be too small.